### ISSUE TITLE
Sample code

### ISSUE THREAD

**Issue by johngoren** [bolein/better-apk-expansion#17]
Wondering if it's possible for the apx_sample directory to include a complete ELI5 example of how the Better APK Expansion software would be implemented. 

And as Better APK seems to be the semi-official go-to library for anyone who is not 31337 enough to hack the official, intimidating Play-APK-Expansion library into shape, it would so be great for Better APK to be a breeze to set up.

I might be confused but I believe the current "apkx_sample" directory, rather than providing an example of how to use Better, is left over from Google's example for the original downloader library and does not use Better APK. Right?

Without a complete example, it's been tricky to figure out how to get this library working. 

Is there a working example out there from another app that we could just bundle in a sample directory, as an easy to follow recipe?

**Comment by vadymhimself** [comment]
You are right, the example is from the original library. Unfortunately, I don't have enough time to fulfill your request, but I would appreciate if you added an example via pull request once you figure out how to use it ;)

**Comment by johngoren** [comment]
Thanks. I have struggled so long with OBB downloads that I can't turn back now! I'll submit a pull request once I have patiently followed your directions and come up with an example.

**Comment by vadymhimself** [comment]
Many thanks, @johngoren! Please be wary of the known issues such as #12.

P.S. OBB system is a mess!

**Comment by johngoren** [comment]
I think for my sanity I'm just going to weather the storm of the transition to giant AABs and for now rely on a vanilla HTTP download from a cloud.
